schemas/org.gnome.shell.extensions.pop-shell.gschema.xml

```diff
@@ -174,5 +174,17 @@
             <default><![CDATA[['<Primary>Right','<Primary>l']]]></default>
             <summary>Swap window right</summary>
         </key>
+        <key type="u" name="log-level">
+            <default>3</default>
```

Comment:
I think we should default to OFF. People can choose what the log when they're actively debugging it.

Reply:
Defaulted it to OFF